app/services/aptrust/aptrust_uploader.rb

```diff
@@ -1,7 +1,67 @@
 # frozen_string_literal: true
 
+require 'zip'
 require_relative './aptrust'
 
+class Aptrust::TarBag
```

Comment:
@ssciolla -- Here's the class wrapper for system tar. Note that it has a dependency on Deepblue::DiskUtilitiesHelper, but this is only to generate a list of files.